opentelemetry-semantic-conventions/src/opentelemetry/semconv/metrics/__init__.py

```diff
@@ -18,17 +18,47 @@
 class MetricInstruments:
 
     HTTP_SERVER_DURATION = "http.server.duration"
+    """
```

Comment:
Might be a dumb question but were these metrics semantic conventions changes in this pr done manually? If we want updating semconv to 1.21.0 to be "complete", we should probably generate these similar to how we did span attributes and resource attributes, which we can probably leave to a separate pr. Is my understanding correct?

Reply:
I think you added them manually a while back - https://github.com/open-telemetry/opentelemetry-python/pull/2976

Let me see if I can enable auto-generation for metric names.

Reply:
done! ptal